gRPC example project is failing on GitHub Windows agent

## Software versions

* **OS**: Windows
* **Consumer Pact library**: v2.0.0-beta.21
* **Provider Pact library**: N/A
* **Golang Version**: 1.19.x
* **Golang environment**: CI

Example project https://github.com/pact-foundation/pact-plugins/tree/main/examples/gRPC/area_calculator/consumer-go was upgraded to the latest Pact-Go using the following commands:
```
go get github.com/pact-foundation/pact-go/v2
go mod vendor
```
See commit https://github.com/pact-foundation/pact-plugins/commit/5f293ae787a89522cd2f4d56b08c374999c97c71

The CI build is now failing on Windows (https://github.com/pact-foundation/pact-plugins/actions/runs/5206582349/jobs/9393298774) with the following error:
```
==== RUNNING consumer-go
# github.com/pact-foundation/pact-go/v2/internal/native
Error: vendor\github.com\pact-foundation\pact-go\v2\internal\native\message_server.go:505:97: cannot use _Ctype_ulong(i) (value of type _Ctype_ulong) as type _Ctype_ulonglong in variable declaration
Error: vendor\github.com\pact-foundation\pact-go\v2\internal\native\message_server.go:509:95: cannot use _Ctype_ulong(i) (value of type _Ctype_ulong) as type _Ctype_ulonglong in variable declaration
Error: Process completed with exit code 2.
```

--- a/internal/native/message_server.go
+++ b/internal/native/message_server.go
@@ -502,11 +502,11 @@ func (m *Message) GetMessageResponseContents() ([][]byte, error) {
 		}
 
 		// Get Response body
-		len := C.pactffi_sync_message_get_response_contents_length(message, C.ulong(i))
+		len := C.pactffi_sync_message_get_response_contents_length(message, C.size_t(i))
 		if len == 0 {
 			return nil, errors.New("retrieved an empty message")
 		}
-		data := C.pactffi_sync_message_get_response_contents_bin(message, C.ulong(i))
+		data := C.pactffi_sync_message_get_response_contents_bin(message, C.size_t(i))
 		if data == nil {
 			return nil, errors.New("retrieved an empty pointer to the message contents")
 		}

